src/clientdlg.h

```diff
@@ -232,7 +233,6 @@ public slots:
     }
 
     void OnConnectDlgAccepted();
-    void OnDisconnected() { Disconnect(); }
```

Comment:
This introduces a bug: disconnecting server side yields to a state where the UI is updated, but the client isn't.

Reply:
Ok. I believe it was https://github.com/jamulussoftware/jamulus/pull/3372/files#diff-2ee628bb1cdcbf04f484b57ea20ddf172db2dfaeb22214ebfb24ff36718a7509R393

Reply:
Should be fixed now.